src/App.tsx

```diff
@@ -1,10 +1,35 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { wait } from './utils/fetchClient';
+import { SortField } from './types/SortField';
+import { filterTodos } from './utils/helpers.ts/filterTodos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
+import { TodoList } from './components/Todolist';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [filter, setFilter] = useState(SortField.All);
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(() => {
+        setError('Unable to load todos');
+        wait(3000).then(() => setError(''));
```

Comment:
Create a handler that sets an error and then removes it in 3 seconds as you have to do it for all requests
```typescrypt
const handleError = (message: string) => {
  setError(message);
  wait(3000).then(() => setError(''));
};
```